{
	"vulinfoid": "JVNDB-2020-016257",
	"vulinfodata": {
		"title": "LibTIFF におけるバッファエラーの脆弱性",
		"vulinfodescription": {
			"overview": "LibTIFF には、バッファエラーの脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "LibTIFF",
					"productname": "LibTIFF",
					"cpe": {
						"text": "cpe:/a:libtiff_project:libtiff",
						"version": "2.2"
					}
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "Medium",
						"type": "Base"
					},
					"base": "4.3",
					"vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "Medium",
						"type": "Base"
					},
					"base": "5.5",
					"vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
				}
			],
			"impactitem": {
				"description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダ情報および参考情報を参照して適切な対策を実施してください。 "
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "vendor",
					"name": "GitLab",
					"vulinfoid": "enforce (configurable) memory limit in tiff2rgba (165)",
					"url": "https://gitlab.com/libtiff/libtiff/-/merge_requests/165"
				},
				{
					"type": "vendor",
					"name": "GitLab",
					"vulinfoid": "enforce (configurable) memory limit in tiff2rgba (98a254f5)",
					"url": "https://gitlab.com/libtiff/libtiff/-/commit/98a254f5b92cea22f5436555ff7fceb12afee84d"
				},
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2020-35522",
					"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-35522"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2020-35522",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-35522"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "Bug 1932037 (CVE-2020-35522) - CVE-2020-35522 libtiff: Memory allocation failure in tiff2rgba",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1932037"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-119",
					"url": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
					"title": "バッファエラー"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2021-11-18T17:00:58+09:00",
					"description": "[2021年11月18日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2021-11-18T17:00:58+09:00",
		"datelastupdated": "2021-11-18T17:00:58+09:00",
		"datepublic": "2020-12-17T00:00:00+09:00"
	}
}
